src/crepe/api/Event.h

```diff
@@ -88,8 +88,28 @@ class MouseMoveEvent : public Event {
 
 	//! Y-coordinate of the mouse position at the time of the event.
 	int mouse_y = 0;
+
+	// Movement since last event in x
+	int rel_x = 0;
+
+	// Movement since last event in y
+	int rel_y = 0;
 };
 
+/**
+ * \brief Event triggered when the mouse is moved.
+ */
+class MouseScrollEvent : public Event {
+public:
+	//! X-coordinate of the mouse position at the time of the event.
+	int scroll_x = 0;
```

Comment:
Weer misleidende namen. `mouse_x` en `mouse_y` of wat het ook alweer op de MouseMoveEvent klasse was. Deze waardes hebben niks met de scrolbeweging te maken.